crates/dbsp/src/circuit/runtime.rs

```diff
@@ -26,6 +26,7 @@ pub enum Error {
     /// Panic in a worker thread.
     WorkerPanic {
         worker: usize,
+        message: String,
```

Comment:
These changes to `WorkerPanic` will be superseded by my upcoming PR that will add panic information to `WorkerPanic`.